app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,63 @@
+class AnswersController < ApplicationController
+  before_action :user_auth, except: %i[index]
+  before_action :set_answer, only: %i[update destroy]
+  before_action :validate_owner, only: %i[update destroy]
+  before_action :check_question_presence, only: %i[create update]
+
+  def index
+    render json: Answer.all, status: :ok
+  end
+
+  def create
+    answer = Answer.new(answer_params.merge(user_id: @current_user.id, question_id: answer_params[:question_id]))
```

Comment:
you're setting the question_id in the answer_params again using the answer_params[:question_id].

Reply:
hmm right